hibernate-core/src/test/java/org/hibernate/test/cfg/persister/GoofyPersisterClassProvider.java

```diff
@@ -265,12 +265,12 @@ public boolean hasLazyProperties() {
 		}
 
 		@Override
-		public Object load(Serializable id, Object optionalObject, LockMode lockMode, SharedSessionContractImplementor session) {
+		public Object load(Serializable id, Object optionalObject, LockMode lockMode, SharedSessionContractImplementor session, Boolean readOnly) {
```

Comment:
IMO this class should compile without changes

Reply:
Again, the old method is still there (though default implementation is provided):

see the following code changes in EntityPersister:
`	/**
	 * Load an instance of the persistent class.
	 */
	Object load(Serializable id, Object optionalObject, LockMode lockMode, SharedSessionContractImplementor session, Boolean readOnly)
	throws HibernateException;

	default Object load(Serializable id, Object optionalObject, LockMode lockMode, SharedSessionContractImplementor session) {
		return load( id, optionalObject, lockMode, session, null );
	}`

Reply:
Reverted it back. Thanks.